packages/fluentui/react-builder/src/components/Designer.tsx

```diff
@@ -221,6 +223,10 @@ const stateReducer: Reducer<DesignerState, DesignerAction> = (draftState, action
       }
       break;
 
+    case 'ENABLE_VIRTUAL_CURSOR':
+      draftState.enabledVirtualCursor = action.enabledVirtualCursor;
```

Comment:
I would expect once I enable the VC, first item in the design is selected. Both when enabled it for the first time as well as when disabled and enabled again.

Reply:
fixed - focus goes to iframe and keyboard shortcuts start to work. also, tooltip with hint is displayed